AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -62,20 +62,17 @@ struct _ABMeasurementsView: View {
                         .font(Fonts.moderateTitle1)
                         .padding(.bottom, 3)
                     HStack {
-                        Group {
-                            ForEach(streams, id : \.self) { stream in
-                                if let threshold = thresholds.threshold(for: stream) {
-                                    SingleMeasurementView(stream: stream,
-                                                          threshold: threshold,
-                                                          selectedStream: $selectedStream,
-                                                          isCollapsed: $isCollapsed,
-                                                          measurementPresentationStyle: measurementPresentationStyle,
-                                                          isDormant: session.isDormant)
-                                }
+                        ForEach(streams, id : \.self) { stream in
+                            if let threshold = thresholds.threshold(for: stream) {
+                                SingleMeasurementView(stream: stream,
+                                                      threshold: threshold,
+                                                      selectedStream: $selectedStream,
+                                                      isCollapsed: $isCollapsed,
+                                                      measurementPresentationStyle: measurementPresentationStyle,
+                                                      isDormant: session.isDormant)
                             }
+                            Spacer()
                         }
-                        .padding(.horizontal, 8)
-                        .frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
Should AB measurements be on the left side of the session card? I think not but tbh I'm not sure. I think that removing the frame and padding moved them to the left. Also, they seem more condensed than previously. 
![IMG_F07B0AE0675D-1](https://user-images.githubusercontent.com/46443841/142613237-d1452fd0-5ae1-4d66-80be-252727c1a5cb.jpeg)

Reply:
Uh, now I see the `Spacer()` - it's its fault the measurements are on the left side.

Reply:
For comparison I'm attaching a photo of how it looked before: 

![image](https://user-images.githubusercontent.com/46443841/142614374-b0fa148a-c9c5-4a4e-92d4-c5b7632e4788.png)